docs/models/wireless/wirelesslink.md

```diff
@@ -40,3 +40,7 @@ The security cipher used to apply wireless authentication. Options include:
 ### Pre-Shared Key
 
 The security key configured on each client to grant access to the secured wireless LAN. This applies only to certain authentication types.
+
+### Length
```

Comment:
IMO "length" doesn't really make sense here, as there's no physical object. "Distance" feels much more natural. FR #15106 proposes "length" for the field name but its title refers to "distance." I realize renaming the fields essentially requires a find-and-replace on the whole PR, but I think it's worthwhile to use the most appropriate terminology.